client/my-sites/google-my-business/stats/fake-data.js

```diff
@@ -0,0 +1,61 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { random, range } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+const dataSummer = ( prevResult, datum ) => prevResult + datum.value;
+
+const searchData = [
+	{
+		name: translate( 'Direct' ),
+		description: translate(
+			'Customers who find your listing searching for you business name or address'
+		),
+		value: random( 300, 500 ),
+	},
+	{
+		name: translate( 'Discovery' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'oEmbed discovery' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).